### PR TITLE
Development of bids2openminds

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

File: bids2openminds/mapping.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
+MAP_2_EXPERIMENTAL_APPROACHES={
```
Comment (lzehl) [inline bids2openminds/mapping.py:1]: we wanted to directly use openminds python objects for the mappings. S can help with this

File: bids2openminds/mapping.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+MAP_2_SPECIES={
+  "homo sapiens":["@id: https://openminds.ebrains.eu/instances/species/homoSapiens"],
+  "mus musculus":["@id: https://openminds.ebrains.eu/instances/species/musMusculus"],
+  "rattus norvegicus":["@id: https://openminds.ebrains.eu/instances/species/rattusNorvegicus"]
+}
+
+
+sample_types={"cell line", "in vitro differentiated cells", "primary cell", "cell-free sample", "cloning host", "tissue", "whole organisms", "organoid", "technical sample"
```
Comment (lzehl) [inline bids2openminds/mapping.py:139]: ???

**Review by lzehl — COMMENTED** [review]
@Peyman-N is this the latest status? if not please push the latest status so that we can review it already before the next meeting.

**Comment by Peyman-N** [comment]
> @Peyman-N is this the latest status? if not please push the latest status so that we can review it already before the next meeting.

Hi @lzehl 
I am still debugging somethings so there is another push and after that I am going to start the next phase.

**Comment by Peyman-N** [comment]
@lzehl I forgot to tell you yesterday, I have debugged it and it is runnable now, so please review it and let me know.

**Comment by apdavison** [comment]
I've run this with the "ds003" dataset from https://github.com/bids-standard/bids-examples.git and it produces some JSON-LD, so I think we can merge this.

@Peyman-N it's still marked as "Draft" - please can you mark it as ready for review?

**Pull request merged**